pkg/loki/config_wrapper_test.go

```diff
@@ -825,20 +825,17 @@ ingester:
 		// ensure they are all false by default
 		config, _, _ := configWrapperFromYAML(t, minimalConfig, nil)
 		assert.False(t, config.QueryRange.ResultsCacheConfig.CacheConfig.EmbeddedCache.Enabled)
-		assert.False(t, config.QueryRange.ResultsCacheConfig.CacheConfig.EmbeddedCache.Distributed)
 
 		configFileString := `---
 query_range:
   results_cache:
     cache:
       embedded_cache:
```

Comment:
Are we sticking with `embedded_cache` as the alias for fifocache?

Reply:
Last I heard, we were

Reply:
:+1: OK cool!